src/libs/DateUtils.ts

```diff
@@ -18,14 +17,15 @@ import {
     isSameYear,
 } from 'date-fns';
 
-import _ from 'underscore';
 import Onyx from 'react-native-onyx';
+// eslint-disable-next-line you-dont-need-lodash-underscore/throttle
```

Comment:
Do you know if it's possible to configure the rule to allow `throttle`?

Reply:
Oh, actually it's quite simple!

```ts
'you-dont-need-lodash-underscore/throttle': 'off',
```